src/nile/core/account.py

```diff
@@ -297,3 +301,10 @@ def get_counterfactual_address(salt=None, calldata=None, contract="Account"):
         constructor_calldata=calldata,
         deployer_address=0,
     )
+
+
+def set_nile_artifacts_path():
+    """Set path to find Nile's precompiled artifacts."""
+    pt = os.path.dirname(os.path.realpath(__file__)).replace("/core", "")
```

Comment:
Any reason to not import `pt` from `nile.common`?

Reply:
If we use it, I would rename `pt` to `nile_root_path` or something similar.

Reply:
> Any reason to not import pt from nile.common?

Nope, good call!